src/main/java/ned/tasks/Deadline.java

```diff
@@ -0,0 +1,62 @@
+package ned.tasks;
```

Comment:
I like that your packages are in all lower case

Reply:
thanks!